kord-extensions/src/main/resources/translations/kordex/strings.properties

```diff
@@ -68,6 +68,7 @@ commands.error.missingBotPermissions=I don't have the permissions I need to run
 commands.error.user=Unfortunately, **an error occurred** during command processing. Please let a staff member know.
 commands.error.user.sentry.message=Unfortunately, **an error occurred** during command processing. If you'd like to submit information on what you were doing when this error happened, please use the following command\: ```{0}feedback {1} <message>```
 commands.error.user.sentry.slash=Unfortunately, **an error occurred** during command processing. If you'd like to submit information on what you were doing when this error happened, please use the following command\: ```/feedback {0} <message>```
+converters.attachment.signatureType=attachment
```

Comment:
This needs to be added to the English translations file as well.

Reply:
Done, added to `strings_en_GB.properties`